mikupad.html

```diff
@@ -2395,14 +2663,143 @@
+					<br />
+					<div className="hbox">
+						<${InputBox} label="Prefix" type="text" placeholder="\\n"
+							readOnly=${!!cancel} value=${worldInfo.prefix} onValueChange=${(value) => handleWorldInfoAffixChange("prefix", value)}/>
+						<${InputBox} label="Suffix" type="text" placeholder="\\n"
+							readOnly=${!!cancel} value=${worldInfo.suffix} onValueChange=${(value) => handleWorldInfoAffixChange("suffix", value)}/>
+					</div>
+				</${CollapsibleGroup}>
+				<button id="button-wi-new" disabled=${!!cancel} onClick=${handleWorldInfoNew}>New Entry</button>
+			</div>
+			<div className="modal-wi-content">
+				${ !Array.isArray(worldInfo.entries) ? null : worldInfo.entries.map((entry, index) => html`
+				<div class="wi-entry" key=${index}>
+					<div class="wi-entry-controls">
+						<div class="wi-entry-filler" />
+						<div class="wi-entry-name">
+							<${InputBox}
+							label="Entry #${index+1}"
+							type="text"
+							readOnly=${!!cancel}
+							placeholder="Name of this entry"
+							value=${entry.displayName}
+							onValueChange=${(value) => handleWorldInfoChange("displayName",index,value)}
+							/>
+						</div>
+						<div class="wi-entry-buttons">
+							<div class="wi-entry-buttons-container">
+								<button disabled=${!!cancel} onClick=${() => handleWorldInfoMove(index,-1)}>
+									<svg fill="var(--color-light)" height="12" width="12" viewBox="0 0 330 330"><path d="M325.606,229.393l-150.004-150C172.79,76.58,168.974,75,164.996,75c-3.979,0-7.794,1.581-10.607,4.394 l-149.996,150c-5.858,5.858-5.858,15.355,0,21.213c5.857,5.857,15.355,5.858,21.213,0l139.39-139.393l139.397,139.393 C307.322,253.536,311.161,255,315,255c3.839,0,7.678-1.464,10.607-4.394C331.464,244.748,331.464,235.251,325.606,229.393z"/></svg>
+								</button>
+								<button disabled=${!!cancel} onClick=${() => handleWorldInfoDel(index)}>
+									✕
+								</button>
+								<button disabled=${!!cancel} onClick=${() => handleWorldInfoMove(index,1)}>
+									<svg fill="var(--color-light)" height="12" width="12" viewBox="0 0 330 330"><path d="M325.607,79.393c-5.857-5.857-15.355-5.858-21.213,0.001l-139.39,139.393L25.607,79.393 c-5.857-5.857-15.355-5.858-21.213,0.001c-5.858,5.858-5.858,15.355,0,21.213l150.004,150c2.813,2.813,6.628,4.393,10.606,4.393 s7.794-1.581,10.606-4.394l149.996-150C331.465,94.749,331.465,85.251,325.607,79.393z"/></svg>
+								</button>
+							</div>
+						</div>
+						<div class="wi-entry-text">
+							<div class="hbox">
+								<${InputBox}
+									label="Comma Separated RegEx Keys"
+									type="text"
+									readOnly=${!!cancel}
+									value=${entry.keys.join(',')}
+									placeholder="Required to activate entry"
+									onValueChange=${(value) => handleWorldInfoChange("keys",index,value)}
+									/>
+								<${InputBox}
+									label="Search Range"
+									tooltip="0 to disable entry. Currently not accurate to the token count, it will be used as an estimate."

```

Comment:
This first part of the tooltip could be in the label, like how it's done in the sidebar.
```suggestion
									label="Search Range (0 = disabled)"
									tooltip="Currently not accurate to the token count, it will be used as an estimate."

```